webapp/src/extensionsBrowser.tsx

```diff
@@ -425,25 +426,20 @@ export const ExtensionsBrowser = (props: ExtensionsProps) => {
                 </div>
                 <div className="extension-display">
                     <div className="extension-header">
-                        {displayMode == ExtensionView.Search &&
-                            <Button
-                                title={lf("Home")}
-                                label={lf("Home")}
-                                onClick={handleHomeButtonClick}
-                                className="link-button"
-                            />
-                        }
-                        {displayMode == ExtensionView.Tags &&
-
+                        {(displayMode == ExtensionView.Search || displayMode == ExtensionView.Tags) &&
                             <div className="breadcrumbs">
                                 <Button
                                     title={lf("Home")}
                                     label={lf("Home")}
                                     onClick={handleHomeButtonClick}
                                     className="link-button"
                                 />
-                                <span>/</span>
-                                <span>{selectedTag}</span>
+                                {displayMode == ExtensionView.Tags &&
+                                    <>
```

Comment:
sidenote: Huh, I saw this in the last pr but this is a pretty nice feature -- I had assumed it just expanded to div but it actually becomes a [React.Fragment](https://reactjs.org/docs/fragments.html) which doesn't become an element in the DOM and just stays as structuring react side. Good to know.